frontend/test/metabase/scenarios/setup/user_settings.cy.spec.js

```diff
@@ -2,6 +2,26 @@
 import { restore, signInAsNormalUser, USERS } from "__support__/cypress";
 const { first_name, last_name, username: email } = USERS.normal;
 
+const CURRENT_USER = {
```

Comment:
Mocking this entire object is kind of gross but wasn't sure of another way without `cy.intercept`

Reply:
Yeah a bit verbose, but I think it gets the jobs done.

Another approach is to turn on LDAP via the API. It looks like this might work: `cy.request("PUT", "/api/setting"
, {"ldap-enabled": true})`. I haven't tested it though and other fields from that form might be required.

Reply:
@paulrosenzweig Only checking the setting would mean that if you have both LDAP and email users, then email users would not be able to change their password.

If we one day officially allow multi-source login, then we'll need to revisit this - just for reference #11322

Reply:
I was trying to play with Paul's suggestion with `ldap-enabled`. It definitely need some additional fields. Until we figure out what exactly and can it be stubbed, this approach works fine for me.